lib/iris/coords.py

```diff
@@ -1578,6 +1578,28 @@ def bounds(self, bounds):
         self._bounds = bounds
 
 
+class CellMeasures(AuxCoord):
+    """
+    CF Cell Measures coordinate. Sublasses AuxCoord to provide string
+    representation for cube attributes list.
+
+    """
+
+    def __init__(self, points, cf_measures, standard_name=None, long_name=None,
+                 var_name=None, units='1', bounds=None, attributes=None,
+                 coord_system=None):
+
+        AuxCoord.__init__(self, points, standard_name=None,
+                          long_name=None, var_name=None, units='1',
+                          bounds=None, attributes=None, coord_system=None)
```

Comment:
>As for the extra args, there is only the one cf_measure argument which is a CF standard name for what property of the cell we are measuring.

I don't think this is what @dkillick means, I think he meant the fact that you ignore the arguments passed in when calling the base constructor, i.e. you have things like `bounds=None` instead of `bounds=bounds` when calling the `AuxCoord` constructor so you are throwing away anything input to the `CellMeasures` constructor.